features/branch.feature

```diff
@@ -1,61 +1,33 @@
-Feature: Check out a feature branch based on a story from Pivotal Tracker
+Feature: Check out a feature branch based on an issue from Linear
   Background:
-    Given a file named "tmp/global_settings.yml" with "pivotal_tracker_api_key: my_api_key"
+    Given a file named "tmp/global_settings.yml" with "linear_api_key: my_api_key"
 
   Scenario: Checkout a new branch with geordi branch
-    Given my username from git config is "Max Musterman"
-      And my local git branches are: master
+    Given my local git branches are: master
 
     When I run `geordi branch` interactively
-      # I skip the initials prompt
       And I type ""
-    Then the output should contain "Util.run! git, checkout, -b, mm/test-story-12"
+    Then the output should contain "Util.run! git, checkout, -b, testuser/12-test-issue"
 
 
   Scenario: Checkout a new branch with geordi branch from master
-    Given my username from git config is "Max Musterman"
-    And my local git branches are: master
+    Given my local git branches are: master
 
     When I run `geordi branch --from-master` interactively
-      # I skip the initials prompt
       And I type ""
     Then the output should contain "Util.run! git, checkout, master"
-      And the output should contain "Util.run! git, checkout, -b, mm/test-story-12"
-
-
-  Scenario: Checkout a new branch with custom initials
-    Given my local git branches are: master
-
-    When I run `geordi branch` interactively
-      # I enter my custom initials
-      And I type "ab"
-    Then the output should contain "Util.run! git, checkout, -b, ab/test-story-12"
-      And the file "tmp/global_settings.yml" should contain "git_initials: ab"
+      And the output should contain "Util.run! git, checkout, -b, testuser/12-test-issue"
 
 
   Scenario: Checkout an existing branch with geordi branch
-    Given my username from git config is "Max Musterman"
-      And my local git branches are: master, mm/test-story-12
+    Given my local git branches are: master, mm/test-issue-12
 
     When I run `geordi branch` interactively
-      # I skip the initials prompt
       And I type ""
     Then the output should not contain "Util.run! git, checkout, master"
-      And the output should contain "Util.run! git, checkout, mm/test-story-12"
+      And the output should contain "Util.run! git, checkout, mm/test-issue-12"
```

Comment:
I am afraid this test is wrong now, and that is in part due to the weak scenario title. How can the test be green right now, if the branch name does not match the new Linear branch name pattern? Please investigate.

If I got it right from the current code, the scenario should be named "Scenario: If the target branch already exists, do not attempt to create a new one but simply switch to the existing branch". So please:
- Rename the branch to the new branch naming pattern
- Remove L27. While it is correct, it is also misleading.

Reply:
Ah, the test is green because of some fallback logic. However, this test is not about the fallback logic, so it should not occur here.